lib/api/messages.js

```diff
@@ -2121,7 +2128,7 @@ module.exports = (db, server, messageHandler, userHandler, storageHandler, setti
             let data = {
                 from: result.value.from || { name: userData.name, address: userData.address },
                 date,
-                to: result.value.to,
+                to: result.value.to ? result.value.to.filter(toObj => toObj.to !== '') : undefined,
```

Comment:
`toObj` should have properties `name` and `address`, not `to`, or am I reading this wrong?

Reply:
Yes, good catch, should be address